safe-svg.php

```diff
@@ -14,7 +14,7 @@
 
 defined( 'ABSPATH' ) or die( 'Really?' );
 
-require 'lib/vendor/autoload.php';
+require 'vendor/autoload.php';
```

Comment:
Since we aren't committing this file to the repo anymore, I think we need to add a conditional around this so those that are directly checking this repo out don't end up with a fatal error if they haven't run `composer install`.

Something like:
```suggestion
// Try and include our autoloader.
if ( is_readable( __DIR__ . '/vendor/autoload.php' ) ) {
    require __DIR__ . '/vendor/autoload.php';
} else {
    add_action( 'admin_notices', function() { ?>
        <div class="notice notice-error">
            <?php /* translators: %1$s: composer command */ ?>
            <p><?php echo wp_kses_post( sprintf( __( 'You appear to be running a development version of Safe SVG. Please run %1$s in order for things to work properly.', 'safe-svg' ), '<code>composer install</code>' ) ); ?></p>
        </div>
<?php
    } );
    return;
}
```

Reply:
Good call, I've added this in 🙂